src/renderer/dx/DxRenderer.cpp

```diff
@@ -975,7 +975,7 @@ try
     {
         // Enable shader effects if the path isn't empty. Otherwise leave it untouched.
         _terminalEffectsEnabled = value.empty() ? _terminalEffectsEnabled : true;
-        _pixelShaderPath = { value };
+        _pixelShaderPath = value.empty() ? L"" : std::wstring{ value };

```

Comment:
i don't totally get it. a wstring_view that is `.empty()` _is_ `""`.

Reply:
Oh. Yikes. When it was broken in the debugger, it looked like the string was full of random memory. I guess I assumed that the view was empty, but secretly actually filled with garbage.